### ISSUE TITLE
Error message when trying to delete a worker

### ISSUE THREAD

**Issue by motaylorvoz** [SavageLearning/Machete#468]
This is what was reported by Graton. They are unable to delete a worker from a data basis and it results in Error!

**Comment by chaim1221** [comment]
Steps to repro:
1. do to people tab 2. choose a worker, 3. push delete on profile page....results in error

**Comment by chaim1221** [comment]
![image](https://user-images.githubusercontent.com/3819970/59233353-f3071f00-8b9c-11e9-96a6-5b7eff47991b.png)

**Comment by chaim1221** [comment]
So this is by design but I'm not closing it because the error message doesn't give the user any idea what is happening. A worker with assignments or signins cannot be deleted.We can't cascade because of reporting purposes. If the worker was actually signed in or assigned, they should not be deleted. They can be expired, sanctioned, or expelled.

**Comment by chaim1221** [comment]
This has not been fixed and actually it's gotten worse. Now it doesn't even say that there was an error trying to delete the worker.

What we really need to implement is "soft delete."

**Comment by esteban-gs** [comment]
I propose we close this issue. 
This may be a permissions issue. 

When I attempt to recreate with a manager role, when I push delete and accept, it does nothing (record is not deleted), and system does not let me know that I don't have enough permissions. 
![image](https://user-images.githubusercontent.com/26440947/87071424-0edd5a00-c1cf-11ea-954e-853ff72cb40b.png)

When I attempt to recreate with a manager role, delete works.

**Comment by chaim1221** [comment]
yes the "doesn't let me know" is the reason I kept it open, see comment above.

**Comment by chaim1221** [comment]
Unassigning myself as I am not actively working on this. Additionally if it's not blocking centers then it's really a v2 consideration, not a v1 one.
